lib/generate-form-url.js

```diff
@@ -16,7 +17,7 @@ function generateFormUrl (
     access_key: token
   }
   if (externalId) {
-    searchParams = Object.assign({}, searchParams, { externalId })
+    searchParams = Object.assign({}, searchParams, { externalId, preFillFormDataId })
```

Comment:
You will prob need to move this into its own check as people may use pre filled data without an external Id i.e.

```js
if (preFillFormDataId) {
    searchParams = Object.assign({}, searchParams, { preFillFormDataId })
}
```